Integrations/URLHaus/URLHaus.py

```diff
@@ -0,0 +1,493 @@
+
+            ec['DBotScore']['Score'] = dbot_score
+            if dbot_score == 3:
+                ec['domain']['Malicious'] = {
+                    'Vendor': 'URLhaus',
+                    'Description': description
+                }
+
+            ec['URLhaus.Domain(val.Name && val.Name === obj.Name)'] = urlhaus_data
+
+            human_readable = tableToMarkdown(f'URLhaus reputation for {domain}',
+                                             {
+                                                 'URLhaus link': domain_information.get('urlhaus_reference', 'None'),
+                                                 'First seen': first_seen,
+                                             })
+            demisto.results({
+                'Type': entryTypes['note'],
+                'ContentsFormat': formats['json'],
+                'Contents': domain_information,
+                'HumanReadable': human_readable,
+                'HumanReadableFormat': formats['markdown'],
+                'EntryContext': ec
+            })
+        elif domain_information['query_status'] == 'no_results':
+            ec['DBotScore']['Score'] = 0
+
+            human_readable = f'## URLhaus reputation for {domain}\n' \
+                f'No results!'
+
+            demisto.results({
+                'Type': entryTypes['note'],
+                'ContentsFormat': formats['json'],
+                'Contents': domain_information,
+                'HumanReadable': human_readable,
+                'HumanReadableFormat': formats['markdown'],
+                'EntryContext': ec
+            })
+        elif domain_information['query_status'] == 'invalid_host':
+            human_readable = f'## URLhaus reputation for {domain}\n' \
+                f'Invalid domain!'
+
+            demisto.results({
+                'Type': entryTypes['note'],
+                'ContentsFormat': formats['json'],
+                'Contents': domain_information,
+                'HumanReadable': human_readable,
+                'HumanReadableFormat': formats['markdown'],
+                'EntryContext': ec
+            })
+        else:
+            demisto.results({
+                'Type': entryTypes['error'],
+                'ContentsFormat': formats['text'],
+                'Contents': f'Query results = {domain_information["query_status"]}'
+            })
+
+    except Exception:
+        demisto.debug(traceback.format_exc())
+        return_error('Failed getting domain data, please verify the arguments and parameters')
+
+
+def file_command():
+    hash = demisto.args()['hash']
+    hash_type = demisto.args()['hash_type'].lower()
+    if hash_type not in ['md5', 'sha256']:
+        return_error('Only accepting MD5 or SHA256 hash types')
+
+    try:
+        file_information = query_payload_information(hash_type, hash).json()
+
+        if file_information['query_status'] == 'ok' and file_information['md5_hash']:
```

Comment:
Where do we handle the sha256?

Reply:
We don't, it's just a distinction from cases where the status is 'ok' but there is no data;.